packages/components/src/components/hds/application-state/header.hbs

```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if @icon}}
     <div class="hds-application-state__icon">
-      <FlightIcon @color="var(--token-color-foreground-strong)" @name={{@icon}} @size="24" />
+      <Hds::Icon @color="var(--token-color-foreground-strong)" @name={{@icon}} @size="24" />
```

Comment:
blocker: since we flipped the flag, similarly to how we removed all `@isInlineBlock={{false}}` instances, where we don't have an `@isInlineBlock` argument we must add `@isInline={{true}}`.

```suggestion
      <Hds::Icon @color="var(--token-color-foreground-strong)" @name={{@icon}} @size="24" @isInline={{true}} />
```

Otherwise we get misaligned icons, also highlighted by VRT

The proposed changed
![hds-showcase-git-3709-internal-icon-migrationc-9c4900-hashicorp vercel app_components_application-state](https://github.com/user-attachments/assets/4ac75ef6-7336-4fbf-9cd2-f2bbe60c627e)

With `@isInline={{true}}`
![localhost_3000_components_application-state](https://github.com/user-attachments/assets/0773f2aa-de71-499a-a21b-2b094d100a04)

Reply:
We only want to do that in specific locations where using them inline was intentional. It was expected that we might have some visual regressions to clean up, but most vertical alignment is done via flexbox, so we shouldn't see too many issues. We do have an option on the codemod for doing what you're suggesting, but we don't suggest using it.